explore/src/main/scala/explore/proposal/ProposalSubmissionBar.scala

```diff
@@ -79,13 +79,15 @@ object ProposalSubmissionBar:
       ctx              <- useContext(AppContext.ctx)
       isUpdatingStatus <- useStateView(IsUpdatingStatus(false))
       errorMessage     <- useStateView(none[String]) // Submission error message
-      _                <- useLayoutEffectWithDeps((props.proposalStatus.get, props.callId, props.hasObsErrors)):
-                            (ps, _, he) =>
-                              if (he && ps === ProposalStatus.NotSubmitted)
-                                errorMessage.set(
-                                  "One or more observations has an error. See Overview tab for details.".some
-                                )
-                              else errorMessage.set(none) // Reset error message on CfP change
+      _                <-
+        useLayoutEffectWithDeps(
+          (props.proposalStatus.get, props.callId, props.hasUndefinedObservations)
+        ): (ps, _, he) =>
+          if (he && ps === ProposalStatus.NotSubmitted)
+            errorMessage.set(
+              "Proposal cannot be submitted with undefined observations. Define them or mark them as inactive.".some
```

Comment:
Is there any use in pointing them to the Overview tab?  That is, should we put that bit back?  Not suggesting it -- I just don't know the answer.

Reply:
I'm going to link to this message in the shortcut story and ask Andy that very question. :) I guess I could do it before merging.